File: DWPayViewOC/payView/ViewController.m
//
//  ViewController.m
//  payView
//
//  Created by DA WENG on 16/10/21.
//  Copyright © 2016年 DA WENG. All rights reserved.
//

#import "ViewController.h"
#import "DWPayView.h"

@interface ViewController ()
{
    UIScrollView* scrollView;
    
    NSTimer* timer;
    NSInteger seconds;
    DWPayView* payView;
}

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    payView = [[DWPayView alloc]initWithorderFee:@"200" whomToPay:@"某某"];
    
    UIButton* btn = [[UIButton alloc]initWithFrame:CGRectMake(80,self.view.frame.size.height - 160, self.view.frame.size.width-160, 30)];
    [btn setTitle:@"确认付款" forState:UIControlStateNormal];
    btn.backgroundColor = [UIColor colorWithRed:91.0/255.0 green:46.0/255.0 blue:123.0/255.0 alpha:1];
    [btn addTarget:self action:@selector(show) forControlEvents:UIControlEventTouchUpInside];
    [self.view addSubview:btn];
    
    [self show];
}

-(void)show{

  [payView showInView:self.view];
}



- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: DWPayViewSwift/DWPayViewSwift/ViewController.swift
//
//  ViewController.swift
//  DWPayViewSwift
//
//  Created by DA WENG on 2017/2/15.
//  Copyright © 2017年 DA WENG. All rights reserved.
//

import UIKit



class ViewController: UIViewController {
    
var gagag = DWPayView()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
    gagag = DWPayView(orderFee: "200", whomToPay: "hello")
  
        let btn = UIButton(frame: CGRect(x: 60, y: 60, width: 60, height: 60))
        btn.backgroundColor = UIColor.red
        btn.addTarget(self, action: #selector(showww), for: UIControlEvents.touchUpInside)
        self.view .addSubview(btn)
        
    }

   
    
    func showww(){
        gagag.showInView(view: self.view)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

